lib/public/views/Tags/Create/index.js

```diff
@@ -42,7 +43,21 @@ const createTagScreen = (model) => {
                 value: text,
                 oninput: (e) => model.tags.setText(e.target.value),
             }, text),
+            model.session?.access?.includes('admin') && [
+                h('label.form-check-label.f4.mt2', { for: 'mattermost' }, 'Mattermost channels'),
```

Comment:
Could we add a descriptive label above "Mattermost" and "Email" Fields so that is clear to the users what are their tags going to do. Something similar to:
```
Tags should inform below channels and groups on log creation:
Mattermost channels:
<mattermost-field>
Email Groups:
<email-groups-field>
```